### PR TITLE
Herocu

### PR DESCRIPTION
**Pull request by chandleryang76**
Integrating Travis-CI and Heroku.
I created new Heroku account which is sdm2016friends@gmail.com
I've found no way of deployment without secrets.yml, so I added it.
If there are someone who know the way, please tell me.
I want all members to review this code.
Thanks

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,7 +25,8 @@ config/secrets.yml
 /vendor/bundle
 
 # these should all be checked in to normalize the environment:
-Gemfile.lock, .ruby-version, .ruby-gemset
+#Gemfile.lock,
```
Comment (allenzhong) [inline .gitignore]: it should be uncommented

Comment (chandleryang76) [inline .gitignore]: I misunderstood it. I remove comment for Gemfile.lock

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+script:
+- bundle exec rake db:create
+- bundle exec rake db:migrate
+
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: "c2471f4b-ebb1-41a5-b8d5-487f86458829"
```
Comment (allenzhong) [inline .travis.yml]: api_key is being exposed. It is not safe. We need find another way to change it. In the past, we add a new template file (might be named .template.secret.yml) with env variables in somewhere of server. In the processing deployment, template file could be copied to application config folder and named secret.yml. So this file could only be saved in server. But I'm sorry I don't know how to do it in Heroku. I need search.

**Comment by chandleryang76** [comment]
I totally agree with Allen opinions. If api is exposed, it is dangerous.
I need to search how to encrypt api key in next sprint

File: .travis.yml (hunk 2 above)
Comment (allenzhong) [inline .travis.yml]: @chandleryang76 finally I fount out that you've already used travis encrypt command....
`
secure: ....
`

**Comment by allenzhong** [comment]
it's ok to merge right now

**Comment by Isabella930514** [comment]
It's ok

**Comment by JohnnyJ0622** [comment]
It's ok!

**Pull request merged**